### DISCUSSION TITLE
Karavan Embedded in our react or angular application

### DISCUSSION THREAD

**Discussion by ashishgidh** [apache/camel-karavan#695] [Q&A]
Hello, we realize that there was an issue raised last year and there is no existing npm package yet to embedd Karavan into our existing angular or react based application. Is there a workaround to achieve this without an npm package ? What is the best way to incorporate the Karavan features into our web application and still control the camel yaml code generated for further processing ? 

Of course we do not want any iFrame techniques but we want to make Karavan a part of our web application. 

Any feedback is much appreciated for a possible path.

**Comment by mgubaidullin** [comment]
We are developing Karavan as a platform for integrators. It was not developed to be embedded into other applications and that's why relys on backed services (cloud or vscode).
